src/components/ConfigurationStructureDiagram.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+const folderData = [
+    {
+        name: "config",
+        children: [
+            { name: "paper-global.yml", url: "/paper/reference/global-configuration" },
+            { name: "paper-world-defaults.yml", url: "/paper/reference/world-configuration" },
+        ],
+    },
+    {
+        name: "world",
```

Comment:
Change to <world> because it's every world folder, not specific ones only?
```suggestion
        name: "<world>",
```